README.md

```diff
@@ -58,6 +58,25 @@ minio_server_make_datadirs: true
 
 Create directories from `minio_server_datadirs` 
 
+```yaml
+minio_server_cluster_nodes: [ ]
+```
+
+Set a list of nodes to create a [distributed cluster](https://docs.minio.io/docs/distributed-minio-quickstart-guide).
+
+In this mode, ansible will create your server datadirs, but use this list for the server startup.
+
+Example:
+
+```yaml
+minio_server_datadirs:
+  - '/minio-data'
```

Comment:
This needs at least 4 directories, not 1.

Reply:
You need 4 storage targets over the whole cluster, not per server.

Reply:
You're right.